### PR TITLE
Fix github actions cache server being down failing builds

### PR DESCRIPTION
**Pull request by RevoluPowered**

### CODE REVIEW

**Review by akien-mga — APPROVED** [review]

File: .github/workflows/linux_builds.yml [hunk 1]
```diff
@@ -39,6 +39,7 @@ jobs:
             ${{github.job}}-${{env.GODOT_BASE_BRANCH}}-${{github.ref}}-${{github.sha}}
             ${{github.job}}-${{env.GODOT_BASE_BRANCH}}-${{github.ref}}
             ${{github.job}}-${{env.GODOT_BASE_BRANCH}}
+        continue-on-error: true # if the cache load fails continue build (very common issue)
```
Comment (aaronfranke) [inline .github/workflows/linux_builds.yml:42]: Is there a reason we don't do this for all jobs?

Comment (RevoluPowered) [reply .github/workflows/linux_builds.yml:42]: cache is the only optional job

Comment (aaronfranke) [reply .github/workflows/linux_builds.yml:42]: What do you mean? The jobs are things like "linux-editor" and "linux-editor-sanitizers". There are multiple jobs per YML file.

Comment (akien-mga) [reply .github/workflows/linux_builds.yml:42]: Ah indeed, there's a few other jobs which need to same flag on their own cache restore steps.

Comment (RevoluPowered) [reply .github/workflows/linux_builds.yml:42]: ah okay, yes I get what you mean, I missed the caches further down in the yaml.

Comment (RevoluPowered) [reply .github/workflows/linux_builds.yml:42]: fixed

Comment (akien-mga) [reply .github/workflows/linux_builds.yml:42]: There's one more for Linux, it has three jobs.

**Review by akien-mga — APPROVED** [review]

File: .github/workflows/linux_builds.yml [hunk 2]
```diff
@@ -215,6 +217,8 @@ jobs:
...
+
```
Comment (aaronfranke) [inline .github/workflows/linux_builds.yml:221]:
```suggestion
        continue-on-error: true # If the cache fails to load, continue the build (very common issue).
```

Comment (akien-mga) [reply .github/workflows/linux_builds.yml:221]: I'd just remove the comment instead of repeating it 10 times.

Comment (RevoluPowered) [reply .github/workflows/linux_builds.yml]: fixed 🤣

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.4.

**Comment by akien-mga** [comment]
Cherry-picked for 3.3.3.

**Pull request merged**